src/components/Dashboard/Dashboard.jsx

```diff
@@ -17,11 +17,11 @@ class Dashboard extends React.Component {
     }
     componentDidMount() {
         socket.on("gameState", msg => {
-            // this.setState({ gameState: msg });
-            console.log(msg);
-            
             this.props.setGameState(msg);
         });
+        socket.on("messages", msg => {
+            this.props.setMessage(msg);
+        });
```

Comment:
I would consider moving the code in this method out of the component. It doesn't really belong here since it only interacts with redux and not the component itself.